crates/abq_queue/src/queue.rs

```diff
@@ -1637,6 +1666,8 @@ impl GetAssignedRun for ChooseRunForWorker {
             run_id,
             batch_size_hint,
         } = invoke_work;
+        // TODO: thread this through, move strategy to net_protocol
```

Comment:
When I (or someone else) picks this back up, we need to...
1. move `Strategy` into `net_protocol` and into the `InvokeWork` struct so that we can send it over the wire, with batch-size, from the CLI
2. then we need to add it to he CLI